features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -121,6 +125,7 @@ import { AccessTokenAttributeOption } from "../access-token-attribute-option";
 import { ApplicationCertificateWrapper } from "../settings/certificate/application-certificate-wrapper";
 import "./inbound-oidc-form.scss";
 
+
```

Comment:
Do we need this new line

Reply:
Will address this with a follow up PR, due to urgency.